py/sea/commands.py

```diff
@@ -1614,6 +1614,88 @@ def run (self, args, extra):
 
         return self.seainspectCmd.run (args, argv)
 
+class CexGen(sea.LimitedCmd):
```

Comment:
good idea to split it into separate file. The file `commands.py` is getting too large. See for example `yama` command